examples/redux/redux-actions/system.config.js

```diff
@@ -1,14 +1,15 @@
-var angularVersion = '2.0.0-rc.1';
+var angularVersion = '2.0.0-rc.6';
```

Comment:
Since there are some other tweaks going on with this, mind changing this to just angular 2.0.0?